src/mmwtex.c

```diff
@@ -5317,6 +5317,8 @@ flag writeBibliography(vstring bibFile,
               || !strcmp(mid(str2, k, (long)strlen("SCOLION")), "SCOLION")
               || !strcmp(mid(str2, k, (long)strlen("SUBSECTION")), "SUBSECTION")
               || !strcmp(mid(str2, k, (long)strlen("TABLE")), "TABLE")
+              || !strcmp(mid(str2, k, (long)strlen("CRITERION")), "CRITERION")
+              || !strcmp(mid(str2, k, (long)strlen("CRITERIA")), "CRITERIA")
```

Comment:
I have to cite `Criterion of [BourbakiTop1] p. I.2` in https://us.metamath.org/mpeuni/ishmeo.html (as a temporary solution, I used "Proposition"). Since the irregular plurals are aslo added in this list (e.g., scolion/scolia), I also added "criteria".
Do you prefer to put the list in alphabetical order ?

Reply:
> 1. the comment in line 5217 should be updated to include mmhlpa.c;

I don't understand this. Line 5217 of which file ? mmhlpa.c has been updated (see its Line 1133).

Reply:
It is line 5274 ff (I got it wrong, don't know why), and it reads
`          /* **IMPORTANT** Make sure to update mmhlpb.c HELP WRITE BIBLIOGRAPHY
             if new items are added to this list. */
`
Here the wrong file is mentioned.

Reply:
Do you prefer to put the list in alphabetical order ?

I don't see a good reason to deviate from it without necessity.  Since the list is duplicated (in a different form, but essentially the items correspond to each other), you can better check the correctness with the aid of some ordering, keeping the exemptions on a low scale..